recipes/ordpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ordpy" %}
+{% set version = "1.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ordpy-{{ version }}.tar.gz
+  sha256: 45a10bcad068854a4e0f76144d0d395c0fd702ea2ea3dbb469e7501be95ec7ee
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - ordpy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/arthurpessa/ordpy
+  summary: 'A Python package for data analysis with permutation entropy and ordinal networks methods.'
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  doc_url: https://ordpy.readthedocs.io/
+  dev_url: https://github.com/arthurpessa/ordpy
+
+extra:
+  recipe-maintainers:
+    - arthurpessa
+    - hvribeiro
```

Comment:
Is this co-maintainer ok with being listed? Either have them comment in the PR that they are or if you know from offline communications, please confirm that.